db/migrate/20180715114637_create_calendar_events.rb

```diff
@@ -0,0 +1,28 @@
+class CreateCalendarEvents < ActiveRecord::Migration[5.1]
+  def change
+    create_table :calendar_events do |t|
+      t.references :creator
```

Comment:
Sorry, I think this should still be `t.references :user` because otherwise I don't think Rails is smart enough to connect the dots. But we can just put a comment in the model in the CalendarEvents has_one :user association to note that the user refers to the creator of the calendar event.

Reply:
In the end I couldn't guarantee it was working properly, still not sure what the easiest way to verify that the association was working properly. At an SQL level, I believe my foreign key was correct, but I had no idea how to get the rails model to understand it.